scripts/upgrade

```diff
@@ -72,14 +70,10 @@ fi
 #
 # See https://www.dokuwiki.org/plugin:config#protecting_settings
 
-
 # Configuration dedicated to Yunohost (LDAP and admin mainly)
 # Create file if it does not exist
 if [ ! -f "$final_path/conf/local.protected.php" ]; then
-	cp ../conf/local.protected.php $final_path/conf
-
-	# Set the default "admin"
-	ynh_replace_string "__YNH_ADMIN_USER__" "$admin" "$final_path/conf/local.protected.php"
+	touch $final_path/conf/local.protected.php
```

Comment:
To not fail at the line [226](https://github.com/YunoHost-Apps/dokuwiki_ynh/blob/c4ae1aacd065ebfa0ef37b95fb525e80b17b97bc/scripts/upgrade#L226) because the file wouldn't exist.

Reply:
Got it now. Could you change the comment above the `touch` command ?

My proposal

```bash
# Create an empty configuration file if it does not exist
# This file will be overwritten anyway latter in part "# Configuration dedicated to Yunohost (LDAP and admin mainly)"
# It's needed so helper `ynh_backup_if_checksum_is_different` will success to backup the new file
```